.github/RELEASE_CHECKLIST.md

```diff
@@ -0,0 +1,15 @@
+# Release Checklist
+
+Before releasing ensure that you have:
+- Run `.release/changelog.sh <PREVIOUS_RELEASE_TAG> <RELEASE_TAG>` to update CHANGELOG.md to contain all commits since the previous tag. Running without the previous release tag will get all commits.
+- Commit the updated changelog via PR with commit message "Preparing for release <RELEASE_TAG>"
+
+Releasing:
+- Go to: https://github.com/hyperledger/fabric-contract-api-go/releases
+- Select "Draft a new release"
+- Tag the version as vX.X.X. Note to include the character v. The repo uses a 3 number semantic versioning system.
```

Comment:
vX.X.X. sounds a little different to "RELEASE_TAG" - do we need to clarify what is meant? Also you've got an extra full stop after the last X

Reply:
This is me trying to tell them how to format the release tag I think I'll move the explanation of release tag structure to the top and make this use release tag